RPI/test/test_processor.py

```diff
@@ -10,51 +10,106 @@ class TestInit(TestCase):
 
```

Comment:
This file still has byte strings, but I assume we will change the tests any way when we have to final input/output layout.